honeybee/model.py

```diff
@@ -1349,6 +1351,46 @@ def to_dict(self, included_prop=None, triangulate_sub_faces=False):
 
         return base
 
+    def to_hbjson(self, name="unnamed", folder_path=None, indent=None, included_prop=None, triangulate_sub_faces=False):
+        """Writes a Honeybee model to HBJSON.
+
+        Args:
+            name: A text string that will be the name of the HBJSOn. Defaults to "unnamed".
+            folder_path: A text string of path to folder where HBJSOn will be written. Defaults to None.
```

Comment:
You should explain what the `None` means. Same below.

Reply:
Explanation added